docs/layer_reference/generative.rst

```diff
@@ -0,0 +1,26 @@
+.. _generative_layers:
+
+=================
+Generative Layers
+=================
+
+Positional Encoding Layer
+-------------------------
+
+.. autoclass:: returnn.tf.layers.rec.PositionalEncodingLayer
+    :members:
+    :undoc-members:
+
+Range Layer
+----------
```

Comment:
Length of `-` does not match. Is this relevant? At least inconsistent.

Reply:
It is not relevant, but should be fixed